packages/expo-analytics-amplitude/src/Amplitude.ts

```diff
@@ -30,7 +30,7 @@ export function initialize(apiKey: string): Promise<void> {
   return ExpoAmplitude.initialize(apiKey);
 }
 
-export function setUserId(userId: string): Promise<void> {
+export function setUserId(userId: string | null): Promise<void> {
```

Comment:
Ensure this works on native, otherwise add an invariant for native.

Reply:
according to the docs it does, see "Logging Out and Anonymous Users" on https://developers.amplitude.com/?objc--ios#setting-custom-user-ids